Orange/widgets/widget.py

```diff
@@ -316,6 +320,12 @@ def save_graph(self):
             return
         saveplot.save_plot(graph_obj, self.graph_writers)
 
+    def __copy_to_clipboard(self):
+        graph_obj = getdeepattr(self, self.graph_name, None)
+        if graph_obj is None:
+            return
+        ClipboardFormat.write_image(None, graph_obj)
```

Comment:
With which visualization? All? Perhaps you should ask @VesnaT. Linear projection report used to fail on Os X in a similar call, and she fixed it.